crates/context_aware_config/src/api/dimension/handlers.rs

```diff
@@ -33,7 +33,7 @@ async fn create(
 ) -> superposition::Result<HttpResponse> {
     let DbConnection(mut conn) = db_conn;
 
-    if req.priority <= 0 {
+    if req.priority == 0 {
```

Comment:
Why did clippy fix this - is it because priority is `uint` ?

Reply:
This was not something clippy automatically fixed, clippy threw an error instead:

```text
error: this comparison involving the minimum or maximum element for this type contains a case that is always true or always false
  --> crates/context_aware_config/src/api/dimension/handlers.rs:36:8
   |
36 |     if req.priority <= 0 {
   |        ^^^^^^^^^^^^^^^^^
   |
   = help: because `0` is the minimum value for this type, the case where the two sides are not equal never occurs, consider using `req.priority == 0` instead
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#absurd_extreme_comparisons
   = note: `#[deny(clippy::absurd_extreme_comparisons)]` on by default
```

Considering that it's a `u16`, I changed the comparison to `==`.